.circleci/config.yml

```diff
@@ -33,6 +33,18 @@ commands:
           fail_only: true
           only_for_branches: main
           webhook: '${SLACK_WEBHOOK}'
+  install_yq:
```

Comment:
Minor: I would include this into the `install_sdk` command and rename it to something like `install_prerequisites`.

Reply:
I introduced instead, what do you think?

Reply:
Don't have a strong opinion, I'm fine either way. The two approaches are not mutually exclusive. You can define this and create `install_prerequisites` where you use it. I'll leave it up to you.